src/main/java/com/booster/server/CreateVocabularyEntryInput.java

```diff
@@ -0,0 +1,11 @@
+package com.booster.server;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
```

Comment:
I think @Builder generates unnecessary code (you can see it in the compiled class version). The same functionality can be reached with @Setter and @Accessors(chain=true) annotations.

Reply:
Agreed. Fixed